lib/chronic_duration.rb

```diff
@@ -2,6 +2,18 @@
 
 module ChronicDuration
 
+  HOURS_IN_DAY  = 8
+  DAYS_IN_WEEK  = 5
```

Comment:
The original gem, on its current master branch, uses a class configuration variable:

https://github.com/henrypoydar/chronic_duration/blob/master/lib/chronic_duration.rb#L34-L36

This seems to work out all the other numbers (5 days for 52 weeks = 260 days / 12 months = 21.6 days)

I do wonder if it's worth looking at the published rubygem again and then reconsidering if we even need this fork.

Reply:
I believe the reason we forked is because we define hours in a day, days in a week, and days in a year differently.  We also changed the formatting when displaying a duration.

Reply:
Also, your math is spot on.  I wonder if the team that originally implemented capacity planning have a reason behind this inconsistency.